docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -100,10 +100,137 @@ audience.
+rates. In this case, the physical characteristic of race impacts the disease. However, 
+this is quite uncommon. 
+
+There isn't any known reason why skin color or other physical characteristics are 
+tied to CVD, so we can dismiss this option for our research question. 
+
+**Option 1b: Race** 
+
+.. image:: race_b_v1.svg
+
+A much more common option is race “leads” to experiences of racism which in turn leads 
+to the disease. We put “leads” in quotes as we don’t want to imply that existing with 
+certain physical characteristics “causes” racism, but rather that racism is the result 
+of years of history defining groups based on physical characteristics. 
+
+This casual pathway is much more common, and seems more applicable to our example with CVD. 
+However, considering the exact pathway here is important. We list a few possible pathways for 
+CVD based on race. Note that these do overlap with ethnicity in some cases. This is 
+because discrimination can impact culture/ethnic practices. These are often 
+rooted in racism. 
+
+- Historical red lining leading to African Americans living in food deserts, having less available green space, or living in neighborhoods with fewer social services
+- Historical racism leading to lower wealth in some communities, reduced access to health insurance, or limited access to quality medical care
+- Microaggressions experienced by non-white individuals leading to elevated blood pressure 
+- Racism in healthcare settings leading to delayed treatment once a person seeks care 
+
+**Option 2a: Ethnicity** 
+
+.. image:: ethnicity_a_v1.svg
+
+The second causal pathway is between ethnicity and disease. Here, there are 
+some compelling arguments for CVD. For example, the mediterranean diet has 
+been shown to lower risk of cardiovascular disease. So ethnic groups that 
+tend to follow this type of diet might have lowered risk of CVD. 
+
+However, a big risk of this causal pathway is that ethnicity can be 
+greatly affected by circumstance and history. 
+
+**Option 2b: Ethnicity** 
+
+.. image:: ethnicity_b_v1.svg
+
+Here is a more complete picture of the same casual diagram which includes history and 
+discrimination. One example of how history impacts ethnicity can be seen by comparing 
+immigrant populations' CVD rates with their counterparts who did not immigrate. As immigrants tend 
+to assimilate to “western” lifestyles, which often involve more processed and 
+fatty foods, they tend to have higher rates of CVD. Here, the outside force is changes in environment and cultural norms, 
+leading to changes in lifestyle and food choices, which in turn to lead to 
+differences in CVD rates. 
+
+Some other examples include: 
+
+- Slavery creating new African American cultural norms separate from African cultural norms 
+- Availability of green space or parks leading to increase exercise in some communities 
+- Food deserts leading to different food practices
```

Comment:
I feel like these might be a bit confusing as first examples, as they all kind of entangle ethnicity with race. Could we maybe start with a more cut-and-dry example like language discrimination?